pkg/prometheus/statefulset.go

```diff
@@ -791,6 +791,7 @@ func makeStatefulSetSpec(
 				AllowPrivilegeEscalation: &boolFalse,
 				ReadOnlyRootFilesystem:   &boolTrue,
 				Capabilities: &v1.Capabilities{
+					Add:  []v1.Capability{"CAP_FOWNER"},
```

Comment:
```suggestion
					// The Thanos sidecar needs the CAP_FOWNER capability because it links block files as hard link.
					Add:  []v1.Capability{"CAP_FOWNER"},
```

Reply:
Done, i have added the comment
Please check it.. thanks !